homeassistant/components/rointe/const.py

```diff
@@ -0,0 +1,39 @@
+"""Constants for the Rointe Heaters integration."""
+
+import logging
+
+from homeassistant.const import Platform
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "rointe"
+DEVICE_DOMAIN = "climate"
+PLATFORMS: list[str] = [Platform.CLIMATE, Platform.SENSOR, Platform.UPDATE]
```

Comment:
If you're going to type this, it would be more accurate to say `list[Platform]`.